feat: Add export of saved data browser filters via `classPreference` settings (#2455)
parse-community/parse-dashboard: README.md

          {
            "name": "email",
            "filterSortToTop": true
          }
        ]
      }
    }
]
```

### Persistent Filters

The `classPreference` setting allows for data browser filters to be saved across sessions. To save the current filters, navigate to *App Settings > Export Class Preferences*.

For example:

```json
"apps": [{
  "classPreference": {
    "_Role": {
      "filters": [{
        "name": "Filter Name",

>         "filter": "[{\"field\":\"objectId\",\"constraint\":\"exists\"}]"

Reviewer: About the filter value, why does its value need to be escaped? Why couldn't we just pass an object that would make it also more readable.

          {
            "name": "email",
            "filterSortToTop": true
          }
        ]
      }
    }
]
```

### Persistent Filters

> `classPreference` allows for filters to be saved across sessions. To save the current filters, navigate to App Settings and click "Export Class Preferences".

Reviewer: ```suggestion
`classPreference` allows for filters to be saved across sessions. To save the current filters, navigate to *App Settings > Export Class Preferences*.

For example:
```

          {
            "name": "email",
            "filterSortToTop": true
          }
        ]
      }
    }
]
```

### Persistent Filters

> `classPreference` allows for filters to be saved across sessions. To save the current filters, navigate to *App Settings > Export Class Preferences*.

Reviewer: ```suggestion
The `classPreference` setting allows for data browser filters to be saved across sessions. To save the current filters, navigate to *App Settings > Export Class Preferences*.
```

For example:

> ```json
> "apps": [
>   {
>     "classPreference": {
>         "_Role": {
>           "filters": [
>             {
>               "name": "Filter Name",
>               "filter": "[{\"field\":\"objectId\",\"constraint\":\"exists\"}]"
>             }
>           ]
>         }
>       }
> ]
> ```

Reviewer: ````suggestion
```json
"apps": [{
  "classPreference": {
    "_Role": {
      "filters": [{
        "name": "Filter Name",
        "filter": "[{\"field\":\"objectId\",\"constraint\":\"exists\"}]"
      }]
    }
  }
}]
```
````

          {
            "name": "email",
            "filterSortToTop": true
          }
        ]
      }
    }
]
```

### Persistent Filters

> The `classPreference` setting allows for data browser filters to be saved across sessions. To save the current filters, navigate to *App Settings > Export Class Preferences*.

Reviewer: ```suggestion
The filters you save in the data browser of Parse Dashboard are only available for the current dashboard user in the current browser session. To make filters permanently available for all dashboard users of an app, you can define filters in the `classPreference` setting.
```

For example:

```json
"apps": [{
  "classPreference": {
    "_Role": {
      "filters": [{
        "name": "Filter Name",
        "filter": [
          {
            "field": "objectId",
            "constraint": "exists"
          }
        ]
      }]
    }
  }
}]
```

> 

Reviewer: ```suggestion

You can conveniently create a filter definition without having to write it by hand by first saving a filter in the data browser, then exporting the filter definition under *App Settings > Export Class Preferences*.

```

The user requires an authenticator app to generate the one-time password. These apps are provided by many 3rd parties and mostly for free.

> If you create a new user by running `parse-dashboard --createUser`, you will be  asked whether you want to enable MFA for the new user. To enable MFA for an existing user,
> run `parse-dashboard --createMFA` to generate a `mfa` secret that you then add to the existing user configuration, for example:

Reviewer: ```suggestion
If you create a new user by running `parse-dashboard --createUser`, you will be  asked whether you want to enable MFA for the new user. To enable MFA for an existing user, run `parse-dashboard --createMFA` to generate a `mfa` secret that you then add to the existing user configuration, for example:
```
